.gitignore

```diff
@@ -250,6 +250,7 @@ paket-files/
 .fake/
 
 # JetBrains Rider
+.editorconfig
 .idea/
 *.sln.iml
```

Comment:
I think this file should not be included in a commit.

Reply:
This adds .editorconfig to the .gitignore so it won't be committed. Sorry I started using Rider recently and this config was missing from the .gitignore